base/TorqueStatorSubsystem.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2011-2023 Texas Torque.
+ *
+ * This file is part of TorqueLib, which is licensed under the MIT license.
+ * For more details, see ./license.txt or write <jus@justusl.com>.
+ */
+package org.texastorque.torquelib.base;
+
+import org.texastorque.torquelib.auto.commands.TorqueExecute;
+
+/**
+ * State based subsystem template.
+ *
+ * @author Justus Languell
+ */
+public abstract class TorqueStatorSubsystem<T extends TorqueState> implements TorqueSubsystem {
```

Comment:
You should rename T to something more descriptive

Reply:
nah https://docs.oracle.com/javase/tutorial/java/generics/types.html